IceCubesApp/Resources/Localization/fr.lproj/Localizable.strings

```diff
@@ -112,8 +108,6 @@
 "settings.system" = "Paramètres système";
 "settings.content.navigation-title" = "Paramètres de contenu";
 "settings.content.boosts" = "Boosts";
-"settings.content.collapse-long-posts" = "Collapse long posts";

```

Comment:
I think there is some problems, you removed a lot of keys instead of translating them?